pkgs/mkpkgs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# NOTE: you'll need to create a arm64 buildx container first
```

Comment:
Looks like I didn't need to do this on my macOS setup

Reply:
We have both types of CI build machines

Reply:
I'll be more specific.